pkg/round/round.go

```diff
@@ -0,0 +1,15 @@
+package round
+
+import (
+	"math"
+)
+
+func RoundP(flo float64, pre uint) float64 {
+	ratio := math.Pow(10, float64(pre))
+	return math.Round(flo*ratio) / ratio
+}
```

Comment:
The `round` package is used to come up with the points on the curve that we try to generate. I had the code for this package including its tests flying around somewhere already.